dev/devicelab/manifest.yaml

```diff
@@ -479,13 +479,6 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios"]
 
-  ios_content_validation_test:
-    description: >
-      Builds an obfuscated app and verifies contents and structure
-    stage: devicelab_ios
-    required_agent_capabilities: ["mac/ios"]
```

Comment:
Oh, it does need a bot which connects to a device. But at this moment, we do not have such bot in prod pool yet.

Reply:
It doesn't need a device.